zksync_sdk/contract_utils.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+try:
+    import importlib.resources as pkg_resources
+except ImportError:
+    # Try backported to PY<37 `importlib_resources`.
+    import importlib_resources as pkg_resources
+from . import contract_abi
+
+zksync_abi_cache = None
+ierc20_abi_cache = None
+
+__all__ = ['zk_sync_abi', 'erc20_abi']
+
+
+def zk_sync_abi():
```

Comment:
Also probably a nitpick, but I think we should be consistent with naming `zksync` or `zk_sync`. I personally prefer the former.